bothub/project/usecases/template_type/creation.py

```diff
@@ -0,0 +1,20 @@
+from .template_type_dto import TemplateTypeDTO
+from bothub.project.models import ProjectIntelligence, TemplateType
+
+class TemplateTypeCreationUseCase:
+
+    def get_repository_info_by_project_uuid(self, project_uuid):
+        info = {"uuid": []}
```

Comment:
The structure of this setup makes it difficult to scale and does not specify which objects the uuids are. I recommend that the structure is similar to:
```py
setup = {
    "repositories": [
        {
            "uuid": "4a5b63af-d5cb-45e1-9843-bd0a84dad95b"
        },
        {
            "uuid": "9427fe19-1662-45b5-8abe-f16a514310e0"
        }
    ]
}
```